### PR TITLE
numa: update cpuset_mems check in interleave and preferred mode

### PR DESCRIPTION
**Pull request by hs0210**
On 8, preferred and interleave mode set all nodes for cpuset.mems cgroup settings
On 9, preferred and interleave mode set blank for cpuset.mems cgroup settings
Both actions have the same effect, both are expected results.

Test Results for `guest_numa_node_tuning.memory_binding_setting`:
```
 (01/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.default_pagesize.mem_mode_strict.single_host_node: PASS (47.41 s)
 (02/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.default_pagesize.mem_mode_interleave.single_host_node: PASS (57.15 s)
 (03/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.default_pagesize.mem_mode_preferred.single_host_node: PASS (56.78 s)
 (04/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.default_pagesize.mem_mode_restrictive.single_host_node: PASS (56.84 s)
 (05/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_strict.single_host_node: PASS (21.80 s)
 (06/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_strict.multiple_host_nodes: PASS (60.29 s)
 (07/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_interleave.single_host_node: PASS (60.17 s)
 (08/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_interleave.multiple_host_nodes: PASS (61.29 s)
 (09/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_preferred.single_host_node: PASS (59.36 s)
 (10/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_preferred.multiple_host_nodes: PASS (19.63 s)
 (11/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_restrictive.single_host_node: PASS (21.61 s)
 (12/12) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_setting.hugepage.mem_mode_restrictive.multiple_host_nodes: PASS (60.72 s)
RESULTS    : PASS 12 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```

Test Results for `guest_numa_node_tuning.auto_memory_nodeset_placement`:
```
 (01/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_strict.vcpu_auto: PASS (45.73 s)                                                                                                                                            
 (02/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_strict.vcpu_static: PASS (52.35 s)                                                                                                                                          
 (03/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_interleave.vcpu_auto: PASS (56.53 s)                                                                                                                                        
 (04/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_interleave.vcpu_static: PASS (46.51 s)                                                                                                                                      
 (05/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_preferred.vcpu_auto: PASS (56.45 s)                                                                                                                                         
 (06/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_preferred.vcpu_static: PASS (44.23 s)                                                                                                                                       
 (07/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_restrictive.vcpu_auto: PASS (48.97 s)                                                                                                                                       
 (08/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_defined.mem_mode_restrictive.vcpu_static: PASS (52.28 s)                                                                                                                                     
 (09/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_none.vcpu_auto: PASS (54.51 s)                                                                                                                                            
 (10/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_none.vcpu_static: PASS (46.23 s)                                                                                                                                          
 (11/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_strict.vcpu_auto: PASS (44.43 s)                                                                                                                                          
 (12/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_strict.vcpu_static: PASS (13.11 s)                                                                                                                                        
 (13/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_interleave.vcpu_auto: PASS (53.31 s)                                                                                                                                      
 (14/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_interleave.vcpu_static: PASS (19.72 s)                                                                                                                                    
 (15/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_preferred.vcpu_auto: PASS (45.43 s)                                                                                                                                       
 (16/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_preferred.vcpu_static: PASS (12.85 s)                                                                                                                                     
 (17/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_restrictive.vcpu_auto: PASS (53.33 s)                                                                                                                                     
 (18/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.without_iothread.nodeset_undefined.mem_mode_restrictive.vcpu_static: PASS (9.46 s)                                                                                                                                    
 (19/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_strict.vcpu_auto: PASS (57.12 s)                                                                                                                                               
 (20/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_strict.vcpu_static: PASS (52.80 s)                                                                                                                                             
 (21/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_interleave.vcpu_auto: PASS (57.66 s)                                                                                                                                           
 (22/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_interleave.vcpu_static: PASS (52.69 s)                                                                                                                                         
 (23/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_preferred.vcpu_auto: PASS (56.23 s)                                                                                                                                            
 (24/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_preferred.vcpu_static: PASS (50.97 s)                                                                                                                                          
 (25/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_restrictive.vcpu_auto: PASS (52.97 s)                                                                                                                                          
 (26/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_defined.mem_mode_restrictive.vcpu_static: PASS (45.76 s)                                                                                                                                        
 (27/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_none.vcpu_auto: PASS (54.39 s)                                                                                                                                               
 (28/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_none.vcpu_static: PASS (46.38 s)                                                                                                                                             
 (29/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_strict.vcpu_auto: PASS (44.56 s)                                                                                                                                             
 (30/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_strict.vcpu_static: PASS (15.55 s)                                                                                                                                           
 (31/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_interleave.vcpu_auto: PASS (47.73 s)                                                                                                                                         
 (32/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_interleave.vcpu_static: PASS (19.29 s)                                                                                                                                       
 (33/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_preferred.vcpu_auto: PASS (46.44 s)                                                                                                                                          
 (34/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_preferred.vcpu_static: PASS (18.16 s)                                                                                                                                        
 (35/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_restrictive.vcpu_auto: PASS (47.16 s)                                                                                                                                        
 (36/36) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.auto_memory_nodeset_placement.with_iothread.nodeset_undefined.mem_mode_restrictive.vcpu_static: PASS (18.81 s)                                                                                                                                      
RESULTS    : PASS 36 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```

### CODE REVIEW

File: libvirt/tests/src/numa/guest_numa_node_tuning/auto_memory_nodeset_placement.py [hunk 1]
```diff
@@ -125,25 +125,28 @@ def verify_cgroup_mem_binding(test_obj):
                                                  surfix_vcpu0)
     cmd_emulator = 'cat %s' % re.escape(emulator_cpuset_mems_cgroup_path)
     cmd_vcpu0 = 'cat %s' % re.escape(vcpu0_cpuset_mems_cgroup_path)
+    numa_info = process.run("numactl -H", ignore_status=True, shell=True).stdout.decode()
```
Comment (chunfuwen) [inline libvirt/tests/src/numa/guest_numa_node_tuning/auto_memory_nodeset_placement.py:128]: stdout_text --> stdout.decode()

Comment (hs0210) [reply libvirt/tests/src/numa/guest_numa_node_tuning/auto_memory_nodeset_placement.py]: Done.
Also, I changed the check measure ,  judging separately based on the difference between 8 and 9 now.
PTAL.

**Review by chunfuwen — APPROVED** [review]
lgtm

**Comment by hs0210** [comment]
@dzhengfy PTAL.

**Comment by hs0210** [comment]
Close as @liang-cong-red-hat has fixed this in a more elegant way.

- https://github.com/autotest/tp-libvirt/pull/5212

- https://github.com/autotest/tp-libvirt/pull/5207

**Pull request closed without merging**